### PR TITLE
Drop preview tag for 15.7 release

### PR DESCRIPTION
**Pull request by AndyGerlicher**

### CODE REVIEW

**Comment by rainersigwald** [comment]
Internal bug: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/605756

**Comment by AndyGerlicher** [comment]
@dotnet-bot test Ubuntu16.04 Build for CoreCLR please

File: src/Build/Microsoft.Build.csproj [hunk 1]
```diff
@@ -19,6 +19,7 @@
     <IsPackable>true</IsPackable>
     <Description>This package contains the $(MSBuildProjectName) assembly which is used to create, edit, and evaluate MSBuild projects.</Description>
     <IncludeSatelliteOutputInPack>false</IncludeSatelliteOutputInPack>
+    <NoWarn>$(NoWarn);NU5104</NoWarn>
```
Comment (rainersigwald) [inline src/Build/Microsoft.Build.csproj]: Comment? I know it's in the commit notes.

**Comment by AndyGerlicher** [comment]
We're running out of 15.7 window. Merging this to prep VS insertion for escrow approval and to ensure no downstream (CLI) issues.

**Pull request merged**